web/source/kmwapi.ts

```diff
@@ -22,36 +22,7 @@
   publishAPI("toNzString", "nzString");
 }());
 
-// Keyboard callbacks
 (function() {
-  let prototype = com.keyman.text.KeyboardInterface.prototype;
-
-  var exportKBCallback = function(miniName: string, longName: string) {
-    prototype[miniName] = prototype[longName];
-  }
-
-  exportKBCallback('KSF', 'saveFocus');
-  exportKBCallback('KBR', 'beepReset');
-  exportKBCallback('KT', 'insertText');
-  exportKBCallback('KR', 'registerKeyboard');
-  exportKBCallback('KRS', 'registerStub');
-  exportKBCallback('KC', 'context');
-  exportKBCallback('KN', 'nul');
-  exportKBCallback('KCM', 'contextMatch');
-  exportKBCallback('KFCM', 'fullContextMatch');
-  exportKBCallback('KIK', 'isKeypress');
-  exportKBCallback('KKM', 'keyMatch');
-  exportKBCallback('KSM', 'stateMatch');
-  exportKBCallback('KKI', 'keyInformation');
-  exportKBCallback('KDM', 'deadkeyMatch');
-  exportKBCallback('KB', 'beep');
-  exportKBCallback('KA', 'any');
-  exportKBCallback('KDC', 'deleteContext');
-  exportKBCallback('KO', 'output');
-  exportKBCallback('KDO', 'deadkeyOutput');
-  exportKBCallback('KIO', 'indexOutput');
-  exportKBCallback('KIFS', 'ifStore');
-  exportKBCallback('KSETS', 'setStore');
-  exportKBCallback('KLOAD', 'loadStore');
-  exportKBCallback('KSAVE', 'saveStore');
+  // DOM-aware KeymanWeb overwrites some of the API functions.
+  com.keyman.text.KeyboardInterface.__publishShorthandAPI();
```

Comment:
This call of `_publishShorthandAPI` ensures that the DOM extensions to `KeyboardInterface` are available where keyboards expect them to be.  This is also necessary to ensure that the DOM-based `registerKeyboard` is in place within KeymanWeb proper, rather than the node-based loader used within `KeyboardProcessor` for unit test execution on Node.